ETH/Android/pocket-chat/app/src/main/java/com/example/myapplication/activity_screen/MessageScreenActivity.java

```diff
@@ -164,7 +164,7 @@ protected void getMessageWith(Integer index) {
         ArrayList<Object> functionParams = new ArrayList<>();
         functionParams.add(new BigInteger(index.toString()));
         try {
-            MessageScreenActivity.this.aionContract.executeConstantFunction("getMessageByIndex", functionParams, null, new BigInteger("50000"), new BigInteger("20000000000"), null, new Function2<PocketError, Object[], Unit>() {
+            MessageScreenActivity.this.ethContract.executeConstantFunction("getMessageByIndex", functionParams, null, new BigInteger("50000"), new BigInteger("20000000000"), null, new Function2<PocketError, Object[], Unit>() {
```

Comment:
gas and gasPrice values are correct? values varies for every blockchain

Reply:
just changed it to the correct gas amount. check new commit